pkg/service/service.go

```diff
@@ -342,13 +344,14 @@ func (s *Service) RegisterL7LBServiceRedirect(serviceName lb.ServiceName, resour
 		return errors.New("proxy port for L7 LB redirection must be nonzero")
 	}
 
-	if logging.CanLogAt(log.Logger, logrus.DebugLevel) {
-		log.WithFields(logrus.Fields{
-			logfields.ServiceName:       serviceName.Name,
-			logfields.ServiceNamespace:  serviceName.Namespace,
-			logfields.L7LBProxyPort:     proxyPort,
-			logfields.L7LBFrontendPorts: frontendPorts,
-		}).Debug("Registering service for L7 proxy port redirection")
+	if s.logger.Enabled(context.Background(), slog.LevelDebug) {
```

Comment:
I'm not sure if the `s.logger.Enabled` check is required. Also, I'm not seeing calls to `SetLogLoggerLevel` in case debug logs are enabled.